lib/evss/reference_data/response_strategy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module EVSS
+  module ReferenceData
+    class ResponseStrategy < Common::RedisStore
+      include Common::CacheAside
+      redis_config_key :reference_data_response
+
+      def cache_or_service(user_uuid, type)
```

Comment:
I think this name is a little confusing, since this method has nothing to do with the service.  I get that's the intended purpose, but used in context I was surprised the block initialized the service because I hadn't seen this method's implementation yet.
I'd propose something like `cache_by_user_and_type` because it makes the purpose clear and explains the arguments.  But there might be a better name.